src/ic-cdk/src/api/management_canister/main/types.rs

```diff
@@ -117,20 +188,63 @@ pub(crate) struct InstallCodeArgumentExtended {
     pub sender_canister_version: Option<u64>,
 }
 
+/// Argument type of [install_chunked_code](super::install_chunked_code).
+#[derive(
+    CandidType, Serialize, Deserialize, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone,
+)]
+pub struct InstallChunkedCodeArgument {
+    /// See [CanisterInstallMode].
+    pub mode: CanisterInstallMode,
+    /// Principal of the canister being installed
+    pub target_canister: CanisterId,
+    /// The canister in whose chunk storage the chunks are stored (defaults to target_canister if not specified)
+    pub storage_canister: Option<CanisterId>,
```

Comment:
The field is called `store_canister`. See here - https://github.com/dfinity/ic/blob/master/rs/types/management_canister_types/src/lib.rs#L2462

Reply:
Hmm I see that its called `storage_canister` in the spec... one of them needs to change